src/main/java/pages/ContactUsPage.java

```diff
@@ -11,7 +11,8 @@
     private WebDriver driver;
     private int timeOut = 5;
 
-    private By contactUsPageTitle = By.xpath("//h1[contains(text(),'Heb je een vraag? Laat het ons weten.')]");
+    private By contactUsPageTitle = By.id("hs_cos_wrapper_module_1556139453617195_");
```

Comment:
Kijk of je deze robuuster kunt krijgen. Deze ID's zijn gegenereerd dus mocht er een nieuwe release plaatst vinden dan gaan deze niet meer werken